cri/ocicni/cni_manager.go

```diff
@@ -87,12 +88,25 @@ func (c *CniManager) GetPodNetworkStatus(netnsPath string) (string, error) {
 		NetNS: netnsPath,
 	}
 
-	ip, err := c.plugin.GetPodNetworkStatus(podNetwork)
+	var err error
+	results, err := c.plugin.GetPodNetworkStatus(podNetwork)
 	if err != nil {
 		return "", fmt.Errorf("failed to get pod network status: %v", err)
 	}
 
-	return ip, nil
+	if len(results) > 0 {
+		// NOTE: only get the first IP of the first network at present
+		// TODO: support multiple network
+		if result, ok := results[0].(*cnicurrent.Result); ok {
```

Comment:
so, is the first one the `*cnicurrent.Result`?

Reply:
Yeah. as comments mentioned, we only get the first IP of the first network at present. :)